src/components/Toggle/Toggle.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import NOOP from "lodash/noop";
+import { useToggleState } from "@react-stately/toggle";
+import { useSwitch } from "@react-aria/switch";
+import { VisuallyHidden } from "@react-aria/visually-hidden";
+import { BASE_TOGGLE_CLASS_NAME } from "./ToggleConstants";
+import ToggleText from "./ToggleText";
+import "./Toggle.scss";
+import { useToggle } from "../../hooks/useToggle";
+
+const Toggle = ({
+  id,
+  componentClassName,
+  isDefaultSelected,
+  isSelected,
+  onChange,
+  value,
+  name,
+  isDisabled,
+  ariaLabel,
+  ariaControls,
+  isHideLabels
+}) => {
+  const { inputProps } = useToggle({
+    id,
+    isDefaultSelected,
+    isSelected,
+    onChange,
+    value,
+    name,
+    isDisabled,
+    ariaLabel,
+    ariaControls
+  });
+
+  const className = classNames(`${BASE_TOGGLE_CLASS_NAME}__toggle`, componentClassName, {
+    [`${BASE_TOGGLE_CLASS_NAME}__toggle--selected`]: inputProps.checked,
+    [`${BASE_TOGGLE_CLASS_NAME}__toggle--not-selected`]: !inputProps.checked,
+    [`${BASE_TOGGLE_CLASS_NAME}__toggle--disabled`]: isDisabled
+  });
+
+  // TODO: ask orr about support for all browsers? MARGIN INLINE
+  // TODO: ask about on and off text
+
+  return (
+    <label htmlFor={id} className={`${BASE_TOGGLE_CLASS_NAME}__wrapper`}>
+      <VisuallyHidden>
+        <input disabled={isDisabled} {...inputProps} />
+      </VisuallyHidden>
+      {isHideLabels ? null : <ToggleText>Off</ToggleText>}
```

Comment:
Off
On

We don't use _t inside monday-ui-react-core?

Reply:
For the open source usage it will be always english.
I will add an  override option and in monday we will use it with (_t("on")) and (_t("off"))